src/lang/en.json

```diff
@@ -2247,6 +2247,11 @@
   "faction.tags.warlike": "Warlike",
   "faction.tags.warlike.description": "There are factions with a military orientation, and then there are factions that just really love killing things. Whether or not this faction has developed sophisticated military resources and techniques, the membership is zealous of battle and has a positive taste for fighting.",
   "faction.tags.warlike.effect": "Once per turn, this faction can roll an additional d10 when making a Force attack.",
+  "entity.settings": "Settings",
+  "settings.hexSystemText": "Hex System Text",
```

Comment:
I have a google doc that I use to keep these updated. If you could prefix these with `misc.` that would be ideal.

Also, since you can also have black holes instead of systems, I've been wanting to rename this in the floating toolbar. Maybe `Hex Content`?

Reply:
I think 'Content' is a bit too broad because it only toggles the text. Maybe `Hex Text` or `Hex Content Text`?

Reply:
`Hex Text` works! I think you're absolutely right.

Reply:
I won't change that in this pull request though, because I can't translate it into the different languages (except German). Are you okay with this?

Reply:
You can change it to `Hex Text` and just remove the translations from the others. I'll add it to the gdrive doc and it'll get translated in time. After that, yeah. This is good to go!

Reply:
Fixed in Irame/sectors-without-number@a4bbed27735a3c3d65f9f6435d542a6858ed03a7